dev-docs/bidders/pangle.md

```diff
@@ -25,6 +26,54 @@ The Pangle Bidding adapter requires setup before beginning. Please contact us at
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description                                           | Example     | Type     |
 |---------------|----------|-------------------------------------------------------|-------------|----------|
-| token         | required | access token                                          | 'aaaa123'   | string   |
+| token         | required | access token,contact Pangle team to get your token    | 'aaaa123'   | string   |
 | appid         | optional | app id (must be used in conjunction with placementid) | '5123400'   | string   |
 | placementid   | optional | placement id (must be used in conjunction with appid) | '912340000' | string   |
+
+Pangle server-side Prebid Server adapter supports `banner`, `video`, `native` media types. But Pangle client-side Prebid.js adapter supports only `banner` media types, doesn't support `video` and `native`.
+
+### Test Parameters
+
+```javascript
+var adUnits = [
+    {
+        code: 'test-div',
+        mediaTypes: {
+            banner: {
+                sizes: [[300, 250]],  // a display size
+            }
+        },
+        bids: [
+           {
+               bidder: "pangle",
+               params: {
+                    placementid: '1234', // string - required
+                    appid: '1234' // string - required
+                }
+           }
+       ]
+    }
+];
+```
+
+### User Sync
+
+Add the following code to enable user sync. Smile Wanted strongly recommends enabling user syncing through iFrames. This functionality improves partners' user match rates and increases the Smile Wanted bid rate and bid price. Be sure to call `pbjs.setConfig()` only once.
```

Comment:
Maybe this is a copy paste error -- should this be "Pangle" rather than "Smile Wanted"?